src/pal/src/include/pal/context.h

```diff
@@ -21,6 +21,8 @@ Module Name:
 #ifndef _PAL_CONTEXT_H_
 #define _PAL_CONTEXT_H_
 
+#include "config.h"
```

Comment:
A nit - this is not needed. The config.h is included in palinternal.h that needs to be always included before this header. We already use a couple of HAVE_xxxx macros in this file.

Reply:
Oops. Fixed now.